extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -0,0 +1,19 @@
+<div id="onSiteNotificationsEntryPoint"
```

Comment:
It's not used anywhere, do we need it? If so, please keep to one convention, you use dashes below (`id="on-site-notifications"`).